pubsub/pubsubcoreapi/pubsub.go

```diff
@@ -106,7 +106,7 @@ func (p *psTopic) WatchMessages(ctx context.Context) (<-chan *iface.EventPubSubM
 		return nil, err
 	}
 
-	ch := make(chan *iface.EventPubSubMessage)
+	ch := make(chan *iface.EventPubSubMessage, 128)
```

Comment:
to get higher stability of https://github.com/berty/berty/blob/master/go/pkg/bertyprotocol/scenario_test.go for 8-nodes and 10-nodes testcases, this value should be higher.